monitor/census.go

```diff
@@ -1319,7 +1338,7 @@ func (cen *censusMetricsCounter) sendSuccess() {
 	stats.Record(cen.ctx, cen.mSuccessRate.M(cen.successRate()))
 }
 
-func SegmentFullyTranscoded(ctx context.Context, nonce, seqNo uint64, profiles string, errCode SegmentTranscodeError) {
+func SegmentFullyTranscoded(ctx context.Context, nonce, seqNo uint64, profiles string, errCode SegmentTranscodeError, orchInfo lpnet.OrchestratorInfo) {
```

Comment:
I think it'd be better to pass `orchInfo` as a pointer here, meaning I'd change:
`orchInfo lpnet.OrchestratorInfo` to `orchInfo *lpnet.OrchestratorInfo`.

If you pass the parameter by value (like you did), all the fields are copied in memory. This struct does not look very big so maybe not such a big issue, but I see we pass this struct by pointer in all other places in the code, e.g. [here](https://github.com/livepeer/go-livepeer/blob/5d9bcaa325d8caa18151b42f50f4f78bf76834d7/server/rpc.go#L116), so I'd be consistent.